units/en/unit3/agentic-rag/agent.mdx

```diff
@@ -484,6 +483,11 @@ print(response['messages'][-1].content)
 </hfoption>
 </hfoptions>
 
+Notice that none of these three agent approaches directly couple memory with the agent. Is there a specific reason for this design choice 🧐?
+* smolagents: Memory is not preserved across different execution runs.
```

Comment:
I'd rephrase this, saying that to preserve memory across runs, you must explicitly state it using `reset=False`